py-polars/tests/unit/operations/test_gather.py

```diff
@@ -378,6 +378,11 @@ def test_gather_group_by_23696(maintain_order: bool) -> None:
     assert_frame_equal(df, expected, check_row_order=maintain_order)
 
 
+def test_gather_invalid_indices_groupby():
```

Comment:
Could you also add the issue number to the test? I.e., `def test_gather_invalid_indices_groupby_24182():`